File: 2 Revalida en C++/5 Funciones/1.cpp
///funcion para evaluar el area
#include <bits/stdc++.h>
using namespace std;
float area(int radio)
{
    float a;
    a=3.14150*(radio*radio);
    return a;
}
int main()
{
    float r1,r2;
    float ass;
    cout<<"ingrese r1:";
    cin>>r1;
    cout<<"ingrese r2:";
    cin>>r2;
    ass=area(r2)-area(r1);
    cout<<"el area de un anillo es ="<<ass;
    return 0;
}

File: 1 Revalida en python/5 Funciones/1.py
#funcion para evaluar el area
def area (radio):
	a=3.14150*(radio*radio)
	return a
#implementaando la funcion en un problema
r1=int(input("ingrese un numero"))
r2=int(input("ingrese un numero"))
ass=area(r2)-area(r1)
print ass